source/tutorial/kotlin.txt

```diff
@@ -143,129 +174,324 @@ Modify the Application
 
 .. procedure::
 
-   .. step:: Add a New Property
+   .. step:: Add a New Property to the Model
 
-      We can add changes now that you have confirmed everything is working as
-      expected. In this tutorial, we have decided that we want to add a
-      "Priority" property to each Item so that we can filter Items by their
-      priorities. The Priority property will be mapped to a PriorityLevel enum
-      to constrain the possible values.
+      Now that you have confirmed everything is working as expected, we can add 
+      changes. In this tutorial, we have decided that we want to add a
+      "priority" property to each Item so that we can filter Items by their
+      priority level. The priority property will be mapped to a ``PriorityLevel`` enum
+      to constrain the possible values, and we will use the ordinal of each 
+      enum to correspond to the priority integer so we can query based on a 
+      numeric priority level later.
       
       To do this, follow these steps:
 
       A. Open the ``build.gradle.kts`` file in Android Studio. 
 
-      #. Within the ``app/java/com.mongodb.app`` folder open the ``Item`` class file.
+      #. Within the ``app/java/com.mongodb.app/domain`` folder, open the ``Item`` class file.
             
-      #. Also, add a PriorityLevel enum above the ``Item`` class:
+      #. Add a ``PriorityLevel`` enum to constrain the possible  
+         values. Also add a ``priority``
+         property to the ``Item`` class, which sets the default priority to 3, 
+         indicating that it is a low-priority todo item:
          
          .. code-block:: kotlin
+            :caption: domain/Item.kt
+            :emphasize-lines: 3-8, 16
+
+            // ... imports
+
+               enum class PriorityLevel() {
+                  Severe, // priority 0
+                  High,   // priority 1
+                  Medium, // priority 2
+                  Low     // priority 3
+               }
+
+               class Item() : RealmObject {
+                  @PrimaryKey
+                  var _id: ObjectId = ObjectId.create()
+                  var isComplete: Boolean = false
+                  var summary: String = ""
+                  var owner_id: String = ""
+                  var priority: Int = PriorityLevel.Low.ordinal
+
+                  constructor(ownerId: String = "") : this() {
+                     owner_id = ownerId
+                  }
```

Comment:
I don't have the client code in front of me as I give this a quick glance, but I think we should show the closing bracket for this `class Item()`. If it's right after the constructor, just include it in the snippet, but if there's a lot of other code between the constructor and the closing bracket, maybe do the `// ... remaining Item() class code` or something along those lines? I'm just thinking of a case where someone tries to use the "copy" button on this snippet to overwrite the existing class code but this is incomplete, but it's not obvious from the snippet that it's incomplete unless you actually pay attention.

Reply:
derp! yep, that was a miss. thanks for catching!